### PR TITLE
{math}[gompi/2022a] lrslib v7.2

### PR DESCRIPTION
**Pull request by migueldiascosta**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by migueldiascosta** [comment]
@boegelbot please test @ generoso

**Comment by migueldiascosta** [comment]
@boegelbot please test @ jsc-zen2

File: easybuild/easyconfigs/l/lrslib/lrslib-7.2-gompi-2022a.eb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+buildopts = 'lrs lrsgmp mplrs CFLAGS="$CFLAGS"'
+
+installopts = 'prefix=%(installdir)s'
+
+sanity_check_paths = {
+    'files': ['bin/%s' % x for x in ['lrs', 'lrsnash']] +
+             ['include/lrslib/lrs%s.h' % h for h in ['driver', 'gmp', 'lib', 'long', 'mp', 'restart']] +
+             ['lib/liblrs.%s' % SHLIB_EXT],
+    'dirs': []
+}
```
Comment (boegel) [inline easybuild/easyconfigs/l/lrslib/lrslib-7.2-gompi-2022a.eb:37]:
```suggestion
}

sanity_check_commands = ["lrsnash --help 2>&1 | grep 'usage.*lrsnash'"]

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3120.skitty.os - Linux RHEL 8.8, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/boegel/02dd3bcdfbd41d1b977736799bd88a01 for a full test report.

**Review by boegel — APPROVED** [review]

**Comment by boegel** [comment]
Going in, thanks @migueldiascosta!

**Pull request merged**